pennylane_lightning/core/src/utils/cuda_utils/cuGates_host.hpp

```diff
@@ -1541,6 +1697,44 @@ template <class PrecisionT> class DynamicGateDataAccess {
          []() -> std::vector<CFP_t> { return cuGates::getToffoli<CFP_t>(); }},
         {"CY", []() -> std::vector<CFP_t> { return cuGates::getCY<CFP_t>(); }},
         {"CZ", []() -> std::vector<CFP_t> { return cuGates::getCZ<CFP_t>(); }},
+        {"I@I",
+         []() -> std::vector<CFP_t> { return cuGates::getIdentity<CFP_t>(); }},
+        {"I@X",
+         []() -> std::vector<CFP_t> { return cuGates::getPauliX<CFP_t>(); }},
+        {"I@Y",
+         []() -> std::vector<CFP_t> { return cuGates::getPauliY<CFP_t>(); }},
+        {"I@Z",
+         []() -> std::vector<CFP_t> { return cuGates::getPauliZ<CFP_t>(); }},
+        {"I@H",
+         []() -> std::vector<CFP_t> { return cuGates::getHadamard<CFP_t>(); }},
+        {"X@I",
+         []() -> std::vector<CFP_t> { return cuGates::getPauliX<CFP_t>(); }},
+        {"X@X",
+         []() -> std::vector<CFP_t> { return cuGates::getIdentity<CFP_t>(); }},
+        {"X@Y", []() -> std::vector<CFP_t> { return cuGates::getXY<CFP_t>(); }},
+        {"X@Z", []() -> std::vector<CFP_t> { return cuGates::getXZ<CFP_t>(); }},
+        {"X@H", []() -> std::vector<CFP_t> { return cuGates::getXH<CFP_t>(); }},
+        {"Y@I",
+         []() -> std::vector<CFP_t> { return cuGates::getPauliY<CFP_t>(); }},
+        {"Y@X", []() -> std::vector<CFP_t> { return cuGates::getYX<CFP_t>(); }},
```

Comment:
Thanks! yea, it works and would reduce lines. But I would prefer to keep it as it is for now since it sounds like Y@X is kind of parametric gate if we're going to pass an arg to it.